server/controller/src/main/java/ai/starwhale/mlops/domain/project/mapper/ProjectMapper.java

```diff
@@ -98,16 +98,16 @@ List<ProjectEntity> listAll(@Param("projectName") String projectName,
     List<ProjectEntity> listRemovedProjects(@Param("projectName") String projectName,
             @Param("ownerId") Long ownerId);
 
-    @Select("select count(*) as count, project_id from model_info"
-            + " where project_id in (${projectIds}) group by project_id")
+    @Select("select count(1) as count, project_id from model_info"
+            + " where project_id in (${projectIds}) and deleted_time = 0 group by project_id")
     List<ObjectCountEntity> countModel(@Param("projectIds") String projectIds);
 
-    @Select("select count(*) as count, project_id from dataset_info"
-            + " where project_id in (${projectIds}) group by project_id")
+    @Select("select count(1) as count, project_id from dataset_info"
+            + " where project_id in (${projectIds}) and deleted_time = 0 group by project_id")
     List<ObjectCountEntity> countDataset(@Param("projectIds") String projectIds);
 
-    @Select("select count(*) as count, project_id from runtime_info"
-            + " where project_id in (${projectIds}) group by project_id")
+    @Select("select count(1) as count, project_id from runtime_info"
```

Comment:
Prefer using `count(*)` 
- They have the same performance
- count(1) can not be used in the left join

ref: https://stackoverflow.com/questions/2710621

Reply:
udpated